src/components/Banners/SiteBanner.tsx

```diff
@@ -23,7 +27,7 @@ export const SiteBanner = () => {
     Cookies.set(text, "viewed", { expires: 7 });
   };
 
-  const variant = mapThemeToVariant[theme] ?? Variant.Info;
+  const variant = mapThemeToVariant[theme?.toLowerCase()] ?? Variant.Info;
```

Comment:
Good call adding the BannerTheme enum, it'd be cool to get all of our banners using it eventually! This is a good stopgap 😎